### DISCUSSION TITLE
Laggy Video

### DISCUSSION THREAD

**Discussion by omnidevnull** [community/events#31] [Community]
Is anyone else have trouble watching the Zapotec video? I'm getting serious lag.

**Comment by queenofcorgis** [comment]
Hi there @omnidevnull - what session are you watching so I can assist? Thanks!

**Reply by JesseStorms** [reply]
I'm experiencing the same issue with both the AI and Community sessions. Audio stutters and buffers.

**Reply by omnidevnull** [reply]
It was a Community video on endangered languages. I don't seem to be having this problem the Codespaces—Git coding fast video.

**Reply by JesseStorms** [reply]
Same here, I guess something mixed up on those tracks

Hopefully we can re-watch that content on demand? (@queenofcorgis)

**Reply by queenofcorgis** [reply]
@JesseStorms All of our core content will be broadcast live and posted on-demand on githubuniverse.com for anyone to watch after the event. Presentations will be available in this repo, post-event.

**Reply by JesseStorms** [reply]
Great, thanks for clarifying!

**Reply by omnidevnull** [reply]
Thank you, @queenofcorgis

**Comment by aarnipavlidi** [comment]
How to open source as a startup: thats lagging :( has different audio. Was curious about this :(

**Reply by aarnipavlidi** [reply]
https://watch.githubuniverse.com/live/dayOneTrackFive

**Reply by kimsauce** [reply]
I'm having the same issue. I'm watching "How to open source as a startup", but the audio is from a completely different session.

**Comment by franzbischoff** [comment]
Oh, it is not just me. Thank you!

**Comment by queenofcorgis** [comment]
Sharing this feedback and your experience with our internal team - thanks everyone for reporting!

**Comment by lucasgonze** [comment]
The live stream is unusable. But that's not a fatal problem - most of us passive consumers can just use the recordings later.

**Comment by jricciardi** [comment]
The stream appears to be back in sync and has been stable for the past two sessions 🎥  - let us know if there are any additional problems! We'll be watching along too for any issues 😄

**Comment by lucasgonze** [comment]
I'm sorry the presentations from today aren't available tonight.
